tools/cloud-build/babysit_tests.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud.devtools import cloudbuild_v1
+from google.cloud.devtools.cloudbuild_v1.types.cloudbuild import Build, ApproveBuildRequest, ApprovalResult
+import time
+import argparse
+
+
+def trig_name(build):
+    return build.substitutions.get("TRIGGER_NAME", "???")
+
+
+def by_name(names):
+    def selector(build):
+        return any(trig_name(build) == n for n in names)
+    return selector
+
+
+SELECTORS = {
+    "all": lambda _: True
+}
+
+
+def get_builds(cb, sha):
+    req = cloudbuild_v1.ListBuildsRequest(
+        project_id="hpc-toolkit-dev",
```

Comment:
could you please parametrize the project ID so it's not into our repo?

Reply:
Added a default `--project` arg, if not set, use `gcloud config get-value project` as default.